src/RepoTokenList.sol

```diff
@@ -364,6 +364,9 @@ library RepoTokenList {
             }
         } else {
             discountRate = discountRateAdapter.getDiscountRate(address(repoToken));

```

Comment:
I noticed that `getDiscountRate` function can revert in two cases, causing the same issue as reported [here](https://github.com/term-finance/yearn-v3-term-vault/issues/56).

Reply:
addressed https://github.com/term-finance/yearn-v3-term-vault/pull/77